openwrt-openwisp-monitoring/files/monitoring.agent

```diff
@@ -26,40 +30,23 @@ if [ "$VERSION" -eq "1" ]; then
   exit 0
 fi
 
-VERIFY_SSL=${VERIFY_SSL:-0}
 INTERVAL=${INTERVAL:-300}
-MONITORED_INTERFACES=${MONITORED_INTERFACES:-*}
 verbose_mode=${verbose_mode:-0}
+TMP_DIR="/tmp/openwisp/monitoring"
 
-if [ -z "$BASE_URL" ]; then
-  logger -s "missing required --url option" \
-         -t openwisp_monitoring \
-         -p daemon.err
-  exit 1
-fi
-
-if [ -z "$UUID" ]; then
-  logger -s "missing required --uuid option" \
-          -t openwisp_monitoring \
-          -p daemon.err
-  exit 1
-fi
-
-if [ -z "$KEY" ]; then
-  logger -s "missing required --key option" \
-         -t openwisp_monitoring \
-         -p daemon.err
-  exit 1
-fi
-
-# Remove double quotes from interfaces
-MONITORED_INTERFACES=$(echo "$MONITORED_INTERFACES" | tr -d '"')
+check_available_memory(){
+  total=$(ubus call system info | grep -m1 total | tr -cd 0-9)
+  available=$(ubus call system info | grep available | tr -cd 0-9)
```

Comment:
can you use jsonfilter to get this? It should be available by default on OpenWRT and should result in being cleaner.